packages/nx/src/project-graph/utils/retrieve-workspace-files.ts

```diff
@@ -155,6 +156,9 @@ export function configurationGlobs(plugins: Array<LoadedNxPlugin>): string[] {
   for (const plugin of plugins) {
     if ('createNodes' in plugin && plugin.createNodes) {
       globPatterns.push(plugin.createNodes[0]);
+    } else {
+      // if there is no glob in the plugin, add an empty glob so that we can still map the globs by index to the plugin
+      globPatterns.push('');
```

Comment:
before going into here, we should filter out the plugins that do not have createNodes already. Since they are not going to be used for creating the project grap